packages/sveltekit/README.md

```diff
@@ -222,37 +222,26 @@ For [row level security](https://supabase.com/docs/learn/auth-deep-dive/auth-row
 ```ts
 // src/routes/profile.ts
 import { supabaseServerClient, withApiAuth } from '@supabase/auth-helpers-sveltekit';
-import type { RequestHandler } from './__types/profile';
+import type { RequestHandler } from './__types/protected-route';
 
 interface TestTable {
 	id: string;
 	created_at: string;
 }
 
 interface GetOutput {
-	user: User;
```

Comment:
To make this example consistent we still need the `user` in the body. Please have a look at line 213. The corresponding `profile.svelte` page expects a `user` prop.